src/test/java/com/icthh/xm/ms/entity/web/rest/FunctionResourceIntTest.java

```diff
@@ -122,6 +122,11 @@ void initLeps(boolean loadData) {
         leps.onRefresh(functionPrefix + "FunctionWithXmEntity$$FUNCTION_WITH_BINARY_RESULT$$tenant.groovy", loadData ? functionWithBinaryResult : null);
         String functionWithNullResult = "return null";
         leps.onRefresh(functionPrefix + "Function$$FUNCTION_WITH_NULL_RESULT$$tenant.groovy", loadData ? functionWithNullResult : null);
+
+        String anonymousFunctionResult = "return [someKey: \"someValue\"]";
+        leps.onRefresh(functionPrefix + "Function$$FUNCTION_WITH_ANONYMOUS_NOT_EXPLICITLY_SET$$tenant.groovy", loadData ? anonymousFunctionResult : null);
```

Comment:
better to change prefix to AnonymousFunction$$<KEY>$$tenant.groovy

Reply:
changed